Subject: appliance: add reboot and netconfig for SUSE
From: Olaf Hering olaf@aepfle.de Tue Sep 14 17:57:43 2021 +0200
Date: Tue Sep 14 20:49:02 2021 +0100:
Git: c0de4de9029c3e483f738a0f80a2c5066c6532db

systemd-sysvinit contains the reboot command, which is used to
properly stop the VM. This was required by other packages, and as a
result always available. Since Leap 15.3 it will not be installed, and
as a result the VM will just panic because /init died.

If the appliance is started with --network, dhclient will run
/usr/sbin/dhclient-script, which in turn may call /sbin/netconfig to
update /etc/resolv.conf. Install sysconfig-netconfig to make sure DNS
resolving actually works.

Signed-off-by: Olaf Hering <olaf@aepfle.de>

--- a/appliance/packagelist.in
+++ b/appliance/packagelist.in
@@ -133,7 +133,9 @@ ifelse(SUSE,1,
   ntfs-3g
   reiserfs
   squashfs
+  sysconfig-netconfig
   systemd
+  systemd-sysvinit
   vim
   xz
 )
